src/Compute/Compute/help/Set-AzVMDiskEncryptionExtension.md

```diff
@@ -95,10 +98,11 @@ $VaultName= "MyKeyVault"
 $KeyVault = Get-AzKeyVault -VaultName $VaultName -ResourceGroupName $RGName
 $DiskEncryptionKeyVaultUrl = $KeyVault.VaultUri
 $KeyVaultResourceId = $KeyVault.ResourceId
-Set-AzVMDiskEncryptionExtension -ResourceGroupName $RGName -VMName $VMName -AadClientID $AADClientID -AadClientSecret $AADClientSecret -DiskEncryptionKeyVaultUrl $DiskEncryptionKeyVaultUrl -DiskEncryptionKeyVaultId $KeyVaultResourceId
+$VolumeType = "All"
+Set-AzVMDiskEncryptionExtension -ResourceGroupName $RGName -VMName $VMName -AadClientID $AADClientID -AadClientSecret $AADClientSecret -DiskEncryptionKeyVaultUrl $DiskEncryptionKeyVaultUrl -DiskEncryptionKeyVaultId $KeyVaultResourceId -VolumeType $VolumeType
 ```
 
-This example enables encryption using Azure AD client ID, and client secret.
+This example enables encryption on a Windows VM using Azure AD client ID, and client secret.
 
 ### Example 4: Enable encryption using Azure AD client ID and client certification thumbprint
```

Comment:
Not related to this change - but I just noticed this example doesn't very clearly demonstrate how to get the certificate on to the VM in the first place.  Might be room for improvement here in the future.